### DISCUSSION TITLE
OpenGL deprecation

### DISCUSSION THREAD

**Discussion by JD31** [pelicanmapping/osgearth#2055] [Support Q&A]
Hello All !
We are currently using OsgEarth for our cross platform application (iOS and Windows) but since few years OpenGL has been deprecated by Apple.
I wonder whether you have a plan for moving out from OpenGL ?
Cheers,
Jérôme.

**Comment by XenonofArcticus** [comment]
Not speaking for Pelican, but AlphaPixel's plan currently is to keep using Apple's OpenGL APIs on iOS and MacOS until Apple forces us to stop using it. We don't expect that to happen for a few years yet. At that point our plan is to build osgEarth for OpenGL ES (as if we were running on an iOS or Android mobile device) and utilize MoltenGL ( https://moltengl.com/moltengl/ ).
